app/controllers/calls_controller.rb

```diff
@@ -0,0 +1,78 @@
+class CallsController < ApplicationController
+
+  after_filter :set_header
+
+  # Public: initiates the call process via a request from twillio
+  #
+  # CallSid - default param from twilio (required)
+  def start
+    response = Twilio::TwiML::Response.new do |r|
+      r.Say 'We need you to connect with your congressperson and senator. We are going to put you in touch with '
+      r.Redirect calls_new_connection_url, method: 'get'
+    end
+
+    render_twiml response
+  end
+
+  # Public: initiates a new outgoing call to congress
+  #
+  # CallSid - default param from twilio (required)
+  def new_connection
+    response = Twilio::TwiML::Response.new do |r|
+      if active_call.target_legislators.any?
+        connection = active_call.create_connection!
+        r.Say 'We will connect you in just a moment. Press star at any time to disconnect from your legislator.'
+        target_number = ENV['FAKE_CONGRESS_NUMBER'] || connection.legislator.phone
+        r.Dial target_number, 'action' => calls_connection_gather_prompt_url, 'hangupOnStar' => true
+      else
+        r.Say 'We don\'t have any targets for you. Please try again later.'
+        r.Hangup
+      end
+    end
+
+    render_twiml response
+  end
+
+  # Public: prompts a user reported response on how the connection preformed
+  #
+  # CallSid - active_call's remote_id from twilio(required)
+  # DialCallSid - dialed call's remote_id from twilio (required)
+  def connection_gather_prompt
+    active_connection = active_call.last_connection
+    active_connection.update(remote_id: params['DialCallSid'])
+    response = Twilio::TwiML::Response.new do |r|
+      r.Gather action: calls_connection_gather_url(connection_id: active_connection.id) do |gather|
+        gather.Say 'How did the senator respond? Press 1 for positively, press 2 for negatively'
+      end
+    end
+
+    render_twiml response
+  end
+
+  # Public: gathers the user response on connection preformance
+  #
+  # CallSid - active_call's remote_id from twilio (required)
+  # Digits - the key pressed user response from twilio (required)
+  # remote_id - remote_for the target connection (required)
+  def connection_gather
+    active_connection = Connection.find(params[:connection_id])
+    active_connection.update(status_from_user: Connection::USER_RESPONSE_CODES[params['Digits']])
```

Comment:
Line is too long. [97/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.